.github/workflows/require-pr-label.yml

```diff
@@ -4,15 +4,14 @@ on:
   pull_request:
     types: [opened, reopened, labeled, unlabeled, synchronize]
 
-permissions:
-  issues: write
-  pull-requests: write
-
 jobs:
   label-dnm:
     name: DO-NOT-MERGE
     if: github.repository_owner == 'python'
     runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
Should this be duplicated for `label-reviews` underneath?
If the permissions are needed for both (i.e. all) jobs, wouldn't it be acceptable to define them at the top level, rather than duplicating them?
I'm also assuming that all jobs in this workflow (including ones that we might add in the future) will edit issue/PR labels and therefore need the `write` permissions.